webapp/client/src/components/FormNavButtons.js

```diff
@@ -0,0 +1,132 @@
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+const Row = styled.div`
+  margin-top: var(--spacing-09);
+`;
+
+const Button = styled.button`
+  padding: 1rem 1.25rem calc(1rem - 4px) 1.25rem;
+
+  font-size: var(--text-base);
+  letter-spacing: var(--tracking-wide);
+  text-decoration: none;
+  color: var(--inverse-text-color);
+  background: var(--link-color);
+  display: inline-block;
+  position: relative;
+  border: none;
+  border-radius: 0;
+  border-bottom: 4px solid var(--link-color);
+
+  &:not(:disabled):not(.disabled):active {
+    background: var(--link-color) !important;
+    border: none !important;
+    border-bottom: 4px solid var(--link-color) !important;
+  }
+
+  &:hover {
+    background: var(--link-hover-color);
+    border: none;
+    border-bottom: 4px solid var(--link-hover-color);
+  }
+
+  &:focus {
+    color: var(--focus-text-color);
+
+    background: var(--focus-color);
+
+    outline: none;
+    box-shadow: none;
+    border: 0;
+    border-bottom: 4px solid var(--focus-border-color);
+  }
+`;
+
+const OutlineButton = styled.button`
+  padding: 1rem 1.25rem calc(1rem - 4px) 1.25rem; /* The calculation subtracts the border-bottom height. We need a border-bottom for the focus state. */
+
+  font-weight: var(--font-bold);
+  letter-spacing: var(--tracking-wide);
+  text-decoration: none;
+  color: var(--link-color);
+
+  background: white;
+  background-clip: padding-box;
+
+  border: 0;
+  border-radius: 0;
+  border: 1px solid var(--border-color);
```

Comment:
It is. I guess that this is just copy-pasted because we had the same lines in `components.css`

Reply:
Yes! :) I'm not making any changes to the styling if I can help it. My goal is to convert everything unchanged with exceptions in those cases where I find bugs or unused styles. If I mix in cleanup work with the conversion it will be very hard to pull in changes that happen on main before the conversion work is finished.